src/pages/DesktopPage.tsx

```diff
@@ -83,11 +83,13 @@ const DesktopPage: React.FC = () => {
                     <IonCard
                       style={{
                         position: "absolute",
+                        margin: "0px",
                         top: 60,
-                        left: 12,
+                        left: 20,
                         // width: "33vw",
                         width: "400px",
-                        height: "90%",
+                        maxWidth: "80%",
```

Comment:
Makes the card slightly smaller on iPad to not overlay the position button on the map